ops/testnet/prod/backend/main.tf

```diff
@@ -117,6 +144,36 @@ module "postgrest" {
   domain                   = var.domain
 }
 
+module "sdk-server" {
+  source                   = "../../../modules/service"
+  region                   = var.region
+  dd_api_key               = var.dd_api_key
+  zone_id                  = data.aws_route53_zone.primary.zone_id
+  execution_role_arn       = data.aws_iam_role.ecr_admin_role.arn
+  cluster_id               = module.ecs.ecs_cluster_id
+  vpc_id                   = module.network.vpc_id
+  private_subnets          = module.network.private_subnets
+  lb_subnets               = module.network.public_subnets
+  internal_lb              = false
+  docker_image             = var.full_image_name_sdk_server
+  container_family         = "sdk-server"
+  container_port           = 8080
+  loadbalancer_port        = 80
+  cpu                      = 256
+  memory                   = 512
```

Comment:
minor: Is this tier enough for SDK Server in prod ?

Reply:
Good question, I am not sure. Will defer to @rhlsthrm